sync/cli/azuredatabricks.py

```diff
@@ -3,6 +3,7 @@
 from sync.cli._databricks import (
     access_report,
     create_prediction,
+    create_recommendation,
```

Comment:
Do we want to add get_recommendation too?

Reply:
Done